src/lib.rs

```diff
@@ -112,7 +114,15 @@ pub async fn start_lemmy_server() -> Result<(), LemmyError> {
     .connect_timeout(REQWEST_TIMEOUT)
     .build()?;
 
+  // Use an in-memory http cache for requests for webfingers & metadata etc...
+  let cache = Cache(HttpCache {
+    mode: CacheMode::Default,
+    manager: MokaManager::new(MokaCache::new(128)),
+    options: None,
+  });
```

Comment:
I dont think a cache with 128 entries is going to make any noticable difference. Lemmy also makes lots of outgoing requests for link previews which might pollute the cache. Also whats your reason for choosing in-memory cache as opposed to disk cache?

In any case I feel like outgoing HTTP requests are only a minor part of the workload, and caching them wouldnt have much of an effect. But if we add this, then it needs to be adjustable via config file and default to no caching for development.